### DISCUSSION TITLE
Version 0.9.7

### DISCUSSION THREAD

**Discussion by jhuckaby** [jhuckaby/Cronicle#493] [General]
- **New Feature:** Schedule page now shows "Last Run" result for each event (success or fail).
    - Saved in state database, survives restarts.
    - Updated in real-time.
    - Click on last run bubbles to jump to latest job detail.
    - Filter events based on last run status.
    - Huge thanks to user @[dheise-rheinenergie-trading-com](https://github.com/dheise-rheinenergie-trading-com) for this feature idea!
- Misc: Removed redundant call to npm install.
- Misc: If no packages were found to restore, don't call npm install a second time.

<img width="379" alt="Screen Shot 2022-05-04 at 9 35 09 PM" src="https://user-images.githubusercontent.com/700765/166864433-a6056484-8510-4c5f-81cf-b7523950e166.png">

<hr /><em>This discussion was created from the release <a href='https://github.com/jhuckaby/Cronicle/releases/tag/v0.9.7'>Version 0.9.7</a>.</em>

**Comment by dheise-rheinenergie-trading-com** [comment]
Sometimes it's the little things that make IT life easier. 😁

**Comment by mikeTWC1984** [comment]
Nice. This might be extended with "Running" badge when job is active, and action to abort the job on click.

**Reply by mcfetz** [reply]
Great idea! Especially the running badge.

**Comment by fpaquet** [comment]
We very much appreciate the new "Last Run" feature.

It could be even more useful, 
when attaching a tooltip to the "Last Run" item, showing the completion date of the last run.

**Reply by jhuckaby** [reply]
@fpaquet Try clicking it 😊

**Comment by fpaquet** [comment]
We have 120 events in "Schedule" Tab. Clicking all of them takes considerable amount of time.
The "Completed" Tab cannot be sorted by event. Anyhow, sorting by event would not help with 380 pages of history.

**Comment by Jacketbg** [comment]
Is there any way to show the old "Status" column? I recently upgraded from an old version and was greeted with the new "Last Run" column. It's not particularly useful in my use case. I have lots of multiplex tasks that need to run simultaneously on different servers and it's really important for me to know the amount of concurrent tasks that are currently running for each event. See the screenshot from the old version:

<img width="309" alt="Screenshot 2024-10-03 at 8 10 24" src="https://github.com/user-attachments/assets/e5601d21-0205-4eed-80b1-3b8b36033273">

Sadly, I had to rollback to an older version because of this change. Hopefully it could be customized and I have missed the option.
Thanks for you hard work!

**Reply by jhuckaby** [reply]
I'm happy to report, this feature is back in v0.9.60!  The column now shows live job counts, OR the last run status if idle.

https://github.com/jhuckaby/Cronicle/releases/tag/v0.9.60

![Screenshot 2024-10-03 at 6 01 00 PM](https://github.com/user-attachments/assets/0c089e0e-d228-4a78-a3dd-b6beaf5598a5)

**Comment by Jacketbg** [comment]
Wow, that was fast! Thank you!
